doc/Overview/Maui/GrapeCity.md

```diff
@@ -0,0 +1,66 @@
+---
+uid: Overview.Maui.GrapeCity
+---
+# .NET MAUI Embedding - GrapeCity
+
+The ComponentOne FlexGrid and Calendar controls from GrapeCity can be used in an Uno Platform application via .NET MAUI Embedding. 
+
+## Installation
+
+In order to use the ComponentOne controls, you first need to install the ComponentOne ControlPanel following [these instructions](https://www.grapecity.com/componentone/docs/maui/online-maui/get-started.html).
+
+## Getting Started
+
+Create a new application using the `unoapp` template, enabling .NET MAUI Embedding. In this case we're going to use the Blank preset to keep things simple.
+
+`dotnet new unoapp -preset blank -maui -o GrapeCityApp`
+
+Remove the .net7.0 (or .net8.0) target framework from both the GrapeCityApp and GrapeCityApp.MauiControls projects.
+
+Next, add a reference to the C1 .NET MAUI nuget packages to the GrapeCityApp.MauiControls library. If you want to use the FlexGrid, add a reference to C1.Maui.Grid; if you want to use the Calendar, add a reference to C1.Maui.Calendar.  
+
+In the `AppBuilderExtensions` class, update the `UseMauiControls` extension method to call either, or both, the `RegisterFlexGridControls` or `RegisterCalendarControls` methods.
+
+```cs
+using C1.Maui.Grid;
+using C1.Maui.Calendar;
+using CommunityToolkit.Maui;
+
+namespace GrapeCityApp;
+
+public static class AppBuilderExtensions
+{
+	public static MauiAppBuilder UseMauiControls(this MauiAppBuilder builder) =>
+		builder.UseMauiCommunityToolkit()
+			.RegisterFlexGridControls()
+      .RegisterCalendarControls();
```

Comment:
This snippet is mixing spaces with tabs and the indentation of `RegisterCalendarControls` isn't aligned with `RegisterFlexGridControls`